aptos-move/block-executor/src/proptest_types/types.rs

```diff
@@ -297,6 +297,13 @@ impl TransactionWrite for ValueType {
     fn set_bytes(&mut self, bytes: Bytes) {
         self.bytes = bytes.into();
     }
+
+    fn from_read_to_modification(&self) -> Option<Self>
```

Comment:
Why does it have to return an Option?

Reply:
leaving to the caller to decide what to do - panic/throw an error/whatever else , if it is deletion.

Reply:
Would be nice if we can clarify that in a comment.